src/Configurator/DockerComposeConfigurator.php

```diff
@@ -125,6 +138,45 @@ private function normalizeConfig(array $config): array
         return $config;
     }
 
+    /**
+     * Finds the Docker Compose file according to these rules: https://docs.docker.com/compose/reference/envvars/#compose_file.
+     */
+    private function findDockerComposeFile(string $rootDir, string $file): ?string
+    {
+        if (isset($_SERVER['COMPOSE_FILE'])) {
+            $files = explode('\\' === \DIRECTORY_SEPARATOR ? ';' : ':', $_SERVER['COMPOSE_FILE']);
```

Comment:
`\DIRECTORY_SEPARATOR` is enought to know if you're working on Linux / Mac or Windows.

However, you can change the separator between files with `COMPOSE_PATH_SEPARATOR` environment variable.

https://docs.docker.com/compose/reference/envvars/#compose_path_separator

Reply:
Thanks for the hint about `COMPOSE_PATH_SEPARATOR`, using it now!

> \DIRECTORY_SEPARATOR is enought to know if you're working on Linux / Mac or Windows.

It's already what is used in Flex and it's what we used to do in Symfony.
It's good enough here because we only care about Windows. We cannot use `PHP_OS_FAMILY` yet because we still support old PHP versions and as it's a Composer plugin we cannot depend on the polyfill.

Reply:
You're welcome.
Thank you too for your amazing job.